Lib/test/libregrtest/main.py

```diff
@@ -87,7 +89,11 @@ def __init__(self, ns: Namespace, _add_python_opts: bool = False):
 
         # Workers
         if ns.use_mp is None:
-            num_workers = 0  # run sequentially
+            if ns.tsan:
+                # For TSAN tests, use number of CPU of the current process.
```

Comment:
I'd add a comment to say why we're ignoring the error for now:

```suggestion
                # For TSAN tests, use number of CPU of the current process.
                #
                # os.process.cpu_count() is new in Python 3.13; 
                # mypy doesn't know about it yet 
```